pkg/secrets/passdriver/passdriver_test.go

```diff
@@ -28,6 +28,9 @@ func setupDriver(t *testing.T) (driver *Driver, cleanup func()) {
 	err = driver.gpg(context.TODO(), nil, nil, "--batch", "--passphrase", "--quick-generate-key", "testing@passdriver")
 	require.NoError(t, err)
 
+	err = driver.gpg(context.TODO(), os.Stdin, os.Stderr, "--batch", "--passphrase", "--quick-generate-key", "testing@passdriver")
+	require.NoError(t, err)
+
```

Comment:
Is there a way to test this with a key that has a password?

Reply:
yeah, I was not sure how to approach this. if we could somehow ensure that `pass` is installed i could test it really easily but it was not installed on my fedora machine. Is there a way to dnf install stuff in these tests?

cc/ @edsantiago

Reply:
I think you don't need pass installed for the passdriver to work, only gpg. Maybe look into generating a key that has a password? And maybe a gpg flag to pass in the password when decrypting the key?

Reply:
I don't understand the full context here, but (1) no you can't dnf-install, (2) have you considered `--passphrase-file`?